### DISCUSSION TITLE
Nuxt 3 typescript support

### DISCUSSION THREAD

**Discussion by justageek** [nuxt/nuxt#17644] [General]
So as far as I can tell from the docs Nuxt 3 fully supports typescript, so am I correct there is no longer a need for the https://typescript.nuxtjs.org/ project to be installed?

**Comment by Diizzayy** [comment]
@justageek Indeed, Nuxt 3 provides [first party support for typescript out the box](http://v3.nuxtjs.org/guide/concepts/typescript).

The project you mentioned was required for Nuxt 2 (ie pre Nuxt Bridge)
